soapui/src/main/java/com/eviware/soapui/support/editor/views/xml/source/XmlSourceEditorView.java

```diff
@@ -628,7 +628,7 @@ public void setEditable(boolean enabled) {
 
     @Override
     public int getSupportScoreForContentType(String contentType ) {
-        return contentType.toLowerCase().endsWith("xml")? 2 : 0;
+        return contentType.toLowerCase().contains("xml")? 2 : 0;

```

Comment:
Isn't that dangerous, since it will also include content types that are not necessarily supported?
For example: `application/
vnd.openxmlformats-officedocument. wordprocessingml.document` which is the MIME-Type of a MS Word .docx file.

Reply:
I agree that the current proposal is sufficient, but if the hold-up is because of how open-ended it is, we could tighten it up by checking to see if it ends in "xml" or if it contains "/xml;".  A regular expression such as the following would work: (?:(xml$)|(/xml;.+))     It is unclear to me--is the "WhiteSource Configuration Change" issue still outstanding?

Reply:
How about split by ";" and check it?  

```
text/xml;charset=UTF-8

[0] text/xml  -> endsWith xml -> stop checking
[1] charset=UTF-8

```